manifests/init.pp

```diff
@@ -76,6 +77,7 @@
   Boolean $manage_service                        = $telegraf::params::manage_service,
   Boolean $manage_repo                           = $telegraf::params::manage_repo,
   Boolean $manage_archive                        = $telegraf::params::manage_archive,
+  Boolean $manage_package                        = $telegraf::params::manage_package,
```

Comment:
$manage_package is always true. In that case you can directly assign the value in the init.pp, and we don't need another variable in the params.pp.